azurelinuxagent/common/cgroupapi.py

```diff
@@ -36,7 +36,7 @@
 
 CGROUPS_FILE_SYSTEM_ROOT = '/sys/fs/cgroup'
 CGROUP_CONTROLLERS = ["cpu", "memory"]
-
+_EXTENSION_SLICE_PREFIX = "azure-vmextensions-"
```

Comment:
Thanks for adding this constant. The idea of using CGroupConfigurator._VMEXTENSIONS_SLICE was that if we ever need to update the name, the change should be done in 1 single place. Could you define CGroupConfigurator._VMEXTENSIONS_SLICE in terms of _EXTENSION_SLICE_PREFIX?